microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -534,6 +534,36 @@ To enhance the Kids Mode experience, Microsoft Edge adds a safe keyword to the h
 Kids Mode does not collect data for personalization of the news feed or other Microsoft services.  You may not change the privacy settings for Kids Mode.  Other settings like Windows Defender SmartScreen and diagnostic data are configured according to the profile in which Kids Mode was launched.  For more information about diagnostic data about how you use the browser and Windows Defender SmartScreen, see the [Diagnostic Data](#diagnostic-data) and [SmartScreen](#smartscreen) sections.
 
 
+<!-- ====================================================================== -->
+## Magnify Image<!-- todo: is the menuitem capital I or lowercase?  global in section -->
+
+To use the Magnify Image feature, right-click an image and then select **Magnify Image**, or press the **Ctrl** key twice while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
+
+
+<!-- ------------------------------ -->
+#### AI-based image enhancement
+
+If your browser has WebNN enabled, there's also a button<!-- todo: what's the button label?  can we commit to specifying the button label at this time?  [click] the **Foo** button --> to use AI to enhance the image.  To check whether WebNN is enabled, go to the `edge://gpu` page, and then in the **Graphics Feature Status** section, see whether **WebNN** says **Enabled**:<!-- todo: how to control whether WebNN is enabled? -->
```

Comment:
what's the button label for AI-based image enhance?  
is it settled enough? i think even if in flux, spec'ing a label here would be close enough, can update later if changes
@sjdallst2

Reply:
WebNN can be enabled by going to edge://flags#web-machine-learning-neural-network

Reply:
```
To change the WebNN setting, go to `edge://flags`, 
search for **Enables WebNN API**, and then select 
**Default**, **Enabled**, or **Disabled**.  This setting 
enables the Web Machine Learning Neural Network (WebNN) 
API.
```